.circleci/src/commands/unit_test.yml

```diff
@@ -1,22 +1,8 @@
 description: Runs the test task and stores coverage and junit data
```

Comment:
Are these changes on codeclimate on purpose? I think you accidentally deleted some of @sarahsporck changes?